src/Library/demos/User Info/main.blp

```diff
@@ -0,0 +1,70 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "User Info";
+  description: _("Request user information");
+  margin-top: 48;
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    Label {
+      label: _("What best describes your request?");
+      margin-bottom: 18;
+      styles ["title-4"]
+    }
+
+    Box {
+      margin-bottom: 30;
+      halign: center;
+      spacing: 18;
+
+      Entry entry {
+        placeholder-text: _("Reason");
+      }
+
+      Button button {
+        label: _("Submit");
+        halign: center;
+        styles ["suggested-action"]
+      }
+    }
+
+    Adw.Avatar avatar {
+      size: 80;
+      margin-bottom: 18;
+    }
+
+    ListBox {
+      selection-mode: none;
+      margin-bottom: 18;
+      styles ["boxed-list"]
+
+      Adw.ActionRow {
+        title: _("ID :");
+
+        Label username {
+          label: _("N/A");
+        }
+      }
+
+      Adw.ActionRow {
+        title: _("Name :");
+
+        Label name {
+          label: _("N/A");
+        }
+      }
+
+    }
```

Comment:
Can you do like ASHPD and hide this until we have the information ?

You could use a Revealer for :sunglasses: effect